src/unauthenticated/LoggedOutRoutes.js

```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LandingPage from './LandingPage';
 import NoMatchPage from '../shared-components/NoMatchPage';
+import AuthCallBack from '../utils/authentication/AuthCallback';
+import AuthService from '../utils/authentication/authService';
 
 const LoggedOutRoutes = () => (
   <Switch>
     <Route exact path="/" component={LandingPage} />;
+    <Route exact path="/callback" component={AuthCallBack} />
```

Comment:
Can we add this config to the auth0 setup that is run as part of the initial bootstrapping?

Reply:
Also, can we make this a bit more descriptive than just callback? Maybe `/auth-callback` or `/login-callback` or something?